webpages/js/metaanalyses.js

```diff
@@ -2149,11 +2170,81 @@
     }
   }
 
+  function getFirstNonHiddenExp(paper) {
+    for (var i=0; i<paper.experiments.length; i++) {
+      if (!isHiddenExp(paper.experiments[i].id)) {
+        return i;
+      }
+    }
+
+    return 0;
+  }
+
+  function getNbUnhiddenExpPaper(paper) {
+    var retval = 0;
+
+    paper.experiments.forEach(function (exp) {
+      if (!isHiddenExp(exp.id)) {
+        retval += 1;
+      }
+    });
+
+    return retval;
+  }
+
+  function getNbHiddenExpPaper(paper) {
+    var retval = 0;
+
+    paper.experiments.forEach(function (exp) {
+      if (isHiddenExp(exp.id)) {
+        retval += 1;
+      }
+    });
+
+    return retval;
+  }
+
+  function isHiddenExp(expId) {
+    return currentMetaanalysis.hiddenExperiments.indexOf(expId) !== -1;
+  }
+
+  function unhideExpPaper(e) {
+    var paperIndex = 0;
+
+    while (currentMetaanalysis.papers[paperIndex].id != e.target.dataset.paperId) {
```

Comment:
Can skip this whole check if we instead pass in paperIndex, since that's what we need (we don't use paperId other than to find the index).